client/app/components/dialog-editor-modal-field-template/dialog-editor-modal-field-template.directive.js

```diff
@@ -0,0 +1,13 @@
+(function() {
+  'use strict';
+
+  angular.module('app.components')
+    .directive('dialogEditorModalFieldTemplate', function() {
```

Comment:
If I remember correctly, I had some issues, when I tried to write this as a component. Maybe it'll be possible in current AngularJS version. I'll take a look, thanks